doc/replication.md

```diff
@@ -68,65 +66,62 @@ devices. This web service needs to:
 ### Replication on the Device
 
 From the device side, replication is straightforward. You can replicate from a
-local datastore to a remote database, from a remote database to a local
-datastore, or both ways to implement synchronisation.
+local document store to a remote database, from a remote database to a local
+document store, or both ways to implement synchronisation.
 
 Replications are set up in code on a device. Use `ReplicatorBuilder` to create
 a pre-configured `Replicator` object. Each `Replicator`
 object can be assigned a `Listener` to receive message when replication
 completes or encounters an error.
 
+The replication examples below are 'one-shot' and will run once until
+completion, at which point the local and remote databases will be
+synchronised. [Replication policies](replication-policies.md) allow
+for more complex "policies" to be executed, eg "only replicate when on
+Wifi" or "replicate every 30 minutes".
+
 First we create a simple listener that just sets a CountDownLatch when the
 replication finishes so we can wait for a replication to finish without
 needing to poll:
 
 ```java
-import com.cloudant.sync.event.Subscribe;
```

Comment:
I can be convinced to add them back but I think they just add noise, and user's IDEs will add them automatically.

I try to think of it this way - the code snipped is not giving a fully formed class which will compile on its own, so why do we need to provide imports?